app/react/Templates/components/MetadataTemplate.tsx

```diff
@@ -131,42 +138,50 @@ export class MetadataTemplate extends Component<MetadataTemplateProps> {
             )}
           </div>
 
-          <ShowIf if={!this.props.relationType}>
-            {connectDropTarget(
-              <ul className="metadataTemplate-list list-group">
-                {commonProperties.map((config: any, index: number) => {
-                  const localID = config.localID || config._id;
-                  return (
-                    <MetadataProperty
-                      {...config}
-                      key={localID}
-                      localID={localID}
-                      index={index - commonProperties.length}
-                    />
-                  );
-                })}
-                {this.props.properties.map((config: any, index: number) => {
-                  const localID = config.localID || config._id;
-                  return (
-                    <MetadataProperty
-                      _id={config._id}
-                      type={config.type}
-                      inserting={config.inserting}
-                      key={localID}
-                      localID={localID}
-                      index={index}
-                    />
-                  );
-                })}
-                <div className="no-properties">
-                  <span className="no-properties-wrap">
-                    <Icon icon="clone" />
-                    Drag properties here
-                  </span>
-                </div>
-              </ul>
-            )}
-          </ShowIf>
+          {environment === 'template' && (
```

Comment:
is this to use this form also for relationType form ?
i think there is better ways to accomplish this reusability, like Component composition.\
Example:
FormWithTitle is the base form, accepts children that MetadataForm can pass to create the extended form
relationType just uses the baseForm or an extension to give semantics.

Reply:
Yes, this is probably true.  This was the least intervention we found (as it was already pseudo implemented with some ShowIfs.... we have already touched a lot of code not belonging to this issue.  Do you think this refactor needs to happen now?

Reply:
if you did not needed to touch this, i prefer to be like it was than making a refactor that does not make the code better.